calcom/cal.com
# Render input field on pro accounts

## What does this PR do?



When on a pro account it will default to displaying an input field to specify the number of seats for that event type.

Fixes # (issue)

https://www.loom.com/share/05bb7c3e136047aebdea86b69e48607e

**Environment**: Staging(main branch)

## Type of change




## How should this be tested?



- Go on the pro account and edit an event type
- Go to advanced options and check off the seats option
- The number input should appear
- Go on a free account and edit an event type
- Go to the advanced options and check off the seats option
- The select input should appear

## Checklist



- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
